vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -1,8 +1,20 @@
 module EmsRefresh::SaveInventoryHelper
   def save_inventory_multi(type, klass, parent, hashes, deletes, find_key, child_keys = [], extra_keys = [])
+    return if hashes.nil?
+    # refresh association if developer is using new syntax (alt: dev passes in nil, [], or actual association values)
+    parent.send(type, true) if deletes == true || deletes == false
```

Comment:
it is currently `nil`/`[]`/`[values]`.
This added 2 options. true - do the lookup for me, and false - don't bother.

In both those cases, we want to refresh the association.
Not sure if we want to always refresh the association. I'm hesitant to add any more work that the refresh needs to do.